app/routes/poll-discourse.tsx

```diff
@@ -0,0 +1,73 @@
+import { useFetcher, useLoaderData } from "@remix-run/react"
+import { useEffect, useState } from "react"
+import {
+  fetchBreakingChangesPosts,
+  fetchMainnetSporkPosts,
+  Topic,
+} from "~/cms/utils/fetch-discourse-api"
+
+type LoaderData = {
+  breakingChanges: Topic[]
+  sporks: Topic[]
+}
+
+export async function loader(): Promise<LoaderData> {
+  const breakingChangesPosts = await fetchBreakingChangesPosts()
+  const sporksPosts = await fetchMainnetSporkPosts()
+  return {
+    breakingChanges: breakingChangesPosts,
+    sporks: sporksPosts,
+  }
+}
+
+export default function () {
+  const data = useLoaderData<LoaderData>()
+  const [discourseData, setDiscourseData] = useState(data)
+
+  // Whenever the loader gives us new data(for example, after a form submission), update our `data` state.
+  useEffect(() => setDiscourseData(data), [data])
+
+  const fetcher = useFetcher()
+
+  // Get fresh data every 5 seconds.
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (document.visibilityState === "visible") {
+        fetcher.load("/poll-discourse")
+      }
+    }, 5 * 1000)
```

Comment:
Can we make this timeout a const eg. `const POLLING_INTERVAL = 5 * 1000` and replace it here.